__tests__/run.test.ts

```diff
@@ -210,6 +210,69 @@ test("run() - deploy - Manifiest not provided", async () => {
     expect(coreMock.setFailed).toBeCalledWith('No manifests supplied to deploy');
 });
 
+test("run() - deploy - Manifests provided by new line delimiter", async () => {
```

Comment:
Can we also have one test where only single manifest file is given with no delimiters?